apps/studio/pages/new/v2/[slug].tsx

```diff
@@ -716,40 +713,60 @@ const WizardForm = () => {
-                                            generateStrongPassword={generatePassword}
-                                          />
-                                        }
-                                      >
-                                        <FormControl_Shadcn_>
-                                          <Input
-                                            copy={field.value.length > 0}
-                                            type="password"
-                                            placeholder="Type in a strong password"
-                                            {...field}
-                                            autoComplete="off"
-                                            onChange={async (event) => {
-                                              field.onChange(event)
-                                              form.trigger('dbPassStrength')
-                                              const value = event.target.value
-                                              if (event.target.value === '') {
-                                                await form.setValue('dbPassStrength', 0)
-                                                await form.trigger('dbPass')
-                                              } else {
-                                                await delayedCheckPasswordStrength(value)
-                                              }
-                                            }}
-                                          />
-                                        </FormControl_Shadcn_>
-                                      </FormItemLayout>
-                                    )}
+                                    render={({ field }) => {
+                                      const regex = /^[^@:\/]*$/
+                                      const hasSpecialCharacters =
+                                        field.value.length > 0 && !field.value.match(regex)
+
+                                      return (
+                                        <FormItemLayout
+                                          label="Database Password"
+                                          description={
+                                            <>
+                                              {hasSpecialCharacters && (
+                                                <p className="mb-2">
+                                                  Note: Special symbols need to be{' '}
+                                                  <InlineLink href="https://supabase.com/docs/guides/database/postgres/roles#special-symbols-in-passwords">
+                                                    percent-encoded
+                                                  </InlineLink>{' '}
+                                                  when using your password later in Postgres
+                                                  connection strings
+                                                </p>
```

Comment:
I would think we should be storing these messages more globally. 
We're making it easy for 1 of these to change in another PR by someone else and they won't realize the same logic and content is replicated elsewhere